cypress/e2e/devpool.cy.ts

```diff
@@ -199,7 +199,7 @@ describe("DevPool", () => {
     cy.get('div[id="issues-container"]').children().should("have.length", 2);
     cy.get('[for="priority-bottom"]').click();
     cy.get('div[id="issues-container"]').children().should("have.length", 2);
-    cy.get("#filters-bottom").scrollTo("right");
+    cy.get("#filters-bottom").scrollTo("right", {ensureScrollable: false});
```

Comment:
Doesn't seem to respect prettier rules.

Reply:
Thanks @gentlementlegen, let me do the changes, Could you approve the workflows

Reply:
I think the workflows are working for each commit?

Reply:
> I think the workflows are working for each commit?

I ran it manually